airflow/www/async_dag_loaders.py

```diff
@@ -0,0 +1,202 @@
+    return stringified_dags
+
+
+def _make_dagbag_update(dagbag, previous_keys, collect_done):
+    dagbag_update = {}
+    for k, v in dagbag.items():
+        current_keys = set(copy.deepcopy(list(v.keys())))
+        new_keys = current_keys - previous_keys[k]
+        previous_keys[k] = set() if collect_done else current_keys
+        if new_keys:
+            dagbag_update[k] = (_stringify_dags(v, new_keys) if k == 'dags'
+                                else {x: v[x] for x in new_keys})
+    return dagbag_update
+
+
+def _send_dagbag(dagbag, queue, event_collect_done, event_next_collect):
+    """A thread that sends dags."""
+    dagbag = {
+        'dags': dagbag.dags,
+        'file_last_changed': dagbag.file_last_changed,
+        'import_errors': dagbag.import_errors
+    }
+    previous_keys = defaultdict(set)
+    while True:  # pylint: disable=too-many-nested-blocks
+        try:
+            collect_done = event_collect_done.is_set()
+            dagbag_update = (dagbag, previous_keys, collect_done)
+
+            if dagbag_update or collect_done:
+                queue.put((collect_done, dagbag_update))
+
+            if collect_done:
+                for v in dagbag.values():
+                    v.clear()
+
+                event_collect_done.clear()
+                event_next_collect.set()
+
+            time.sleep(DAGBAG_SYNC_INTERVAL)
+        except Exception:  # pylint: disable=broad-except
+            logging.warning('Dagbag loader sender errors.', exc_info=True)
+
+
+def _create_dagbag(dag_folder, queue):
+    """A process that creates, updates, and sync dagbag in background."""
+    import airflow
+    # pylint: disable=redefined-outer-name,reimported
+    from airflow import configuration
+
+    # TODO(coufon): env vars that are not exposed to subprocesses are invisible here.
+    # In Composer we reload all env vars here, should check whether a similar step may
+    # be needed.
+
+    configuration = importlib.reload(configuration)
+    airflow.configuration = importlib.reload(airflow.configuration)
+    airflow.plugins_manager = importlib.reload(airflow.plugins_manager)
+    airflow = importlib.reload(airflow)
+
+    signal.signal(signal.SIGTERM, _kill_proc)
+    signal.signal(signal.SIGINT, _kill_proc)
+
+    logging.info('Using Asynchronous Dagbag Loader.')
+    dagbag = DagBag(dag_folder)
+    event_collect_done = Event()
+    event_next_collect = Event()
+    thread = threading.Thread(target=_send_dagbag,
+                              args=(dagbag, queue, event_collect_done,
+                                    event_next_collect))
+    thread.daemon = True
+    thread.start()
+    while True:
+        try:
+            event_collect_done.clear()
+            event_next_collect.clear()
+
+            start_time = time.time()
+            dagbag.collect_dags(
+                dag_folder,
+                include_examples=configuration.getboolean('core', 'LOAD_EXAMPLES'))
```

Comment:
`dagbag.collect_dags` handles `include_examples`, so we can skip that here

Reply:
> `dagbag.collect_dags` handles `include_examples`, so we can skip that here

Removed include_examples here.